src/components/typeahead/typeahead.vue

```diff
@@ -0,0 +1,261 @@
+<script>
+    const UID_key = '__pd_ta_uid'
+
+    /* Implementation of lodash.get function */
+    function getProp( object, keys, defaultVal ){
```

Comment:
Would it make sense to make this global to the components so it can be reused?

Reply:
potentially. for now, follow YAGNI?